pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>github-branch-source</artifactId>
             <version>2.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.mylyn.github</groupId>
+            <artifactId>org.eclipse.egit.github.core</artifactId>
+            <version>5.2.0.201812061821-r</version>
+        </dependency>
```

Comment:
You should remove this, yes you can use https://plugins.jenkins.io/github-api/ as it is a transitive dependency :)
```suggestion
```

Reply:
I made the same suggestion and sent an implementation to @lprimak that should work. Waiting for testing results.

Reply:
missed that one